Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebTransactionView.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+struct PayPalWebTransactionView: View {
+
+    @ObservedObject var payPalWebViewModel: PayPalWebViewModel
+
+    var body: some View {
+        ScrollView {
+            VStack {
+                PayPalWebStatusView(status: .approved, payPalViewModel: payPalWebViewModel)
+                ZStack {
+                    Button("\(payPalWebViewModel.intent.rawValue.capitalized) Order") {
+                        Task {
+                            do {
+                                try await payPalWebViewModel.completeTransaction()
+                            } catch {
+                                print("Error capturing order: \(error.localizedDescription)")
+                            }
+                        }
+                    }
+                    .buttonStyle(RoundedBlueButtonStyle())
+                    .padding()
+                    
+                    if payPalWebViewModel.state == .loading {
+                        CircularProgressView()
+                    }
+                }
+                
+                if payPalWebViewModel.state == .success {
+                    PayPalWebResultView(payPalWebViewModel: payPalWebViewModel, status: .completed)
+                }
+            }
+            .onAppear {
+                payPalWebViewModel.resetState()
```

Comment:
One behavior change is that if I navigate back from this view to original screen, I have this weird look.
That's one thing about the view resetting state in the vm on appear. I am thinking about other modules where there are many steps and I have container views. And if I go back and forth, the views with results would be wiped out. 

I have done this in my testing so I can check the results in previous screen. There is also different setup in card vault without purchase flow in demo app where I just display results in a large scrollview and scroll to bottom as results come up. So I think I am reliant on viewModel keeping track of state. I guess then I can always NOT reset the state.


<img src="https://github.com/paypal/paypal-ios/assets/9273272/ce47fccc-8c16-46a4-a097-33fd1b8d4bc8" width="300">

Reply:
Easy, we can rely on the status instead of resetting the state: https://github.com/paypal/paypal-ios/pull/223/commits/94f8f9b51a6e5e640a238996751ddadbcac0134d. If you just want to triple check me but the behavior looks to be the same as before.

Reply:
Thank you for fast fixes! I just want to make sure that I have complete understanding for following this pattern for other modules.

Reply:
I am missing scroll to bottom behavior that I had with vaulting option when the view is longer.
I used this scrolling to bottom a lot in vault without purchase for card payment.

Reply:
We can rely on the change to order (updated with the call to complete transaction) to auto scroll to the bottom. Updated here: https://github.com/paypal/paypal-ios/pull/223/commits/c2785ae8e63b361b7052c57590639683cd54a3a8